lib/flame_on/component.ex

```diff
@@ -139,7 +71,7 @@ defmodule FlameOn.Component do
         socket
       end
 
-    socket = assign(socket, :capture_changeset, changeset)
+    socket = assign(socket, :capture_changeset, Map.put(changeset, :action, :insert))
```

Comment:
Not a "request changes" per se, but this smells a bit to me. Shouldn't this be left to apply_action or something?